src/pages/sprints/days/TimeLine.vue

```diff
@@ -65,19 +34,51 @@
 </template>
 
 <script>
+import { mapActions, mapState } from "pinia";
+
+import { useContentStore } from "@/store/contentStore.js";
+
 export default {
   props: {
-    day: {
-      type    : Object,
-      default : () => { return {}; }
+    index: {
+      type    : Number,
+      default : () => 0
     },
     listTask: {
       type    : Array,
       default : () => []
     },
-    index: {
-      type    : Number,
-      default : () => 0
+    day: {
+      type    : Object,
+      default : () => { return {}; }
+    }
+  },
+  methods: {
+    ...mapActions(useContentStore, ["selectContent"]),
+    changeRouter(router = "", contIndex) {
+      this.selectContent(contIndex);
+      mapState(useContentStore, ["contIndex"]);
```

Comment:
huh

Reply:
i think you are updating the state with the new num 
Then getting the new num from the state 
I think ???